fawltydeps/settings.py

```diff
@@ -100,21 +100,42 @@ def parse_path_or_stdin(arg: str) -> PathOrSpecial:
 
 DEFAULT_IGNORE_UNUSED = {
     # Development tools not meant to be imported
+    # Formatting Tools
+    "autopep8",
     "black",
+    "isort",
+    "pyformat",
+    "yapf",
+    # Linting Tools
+    "bandit",
+    "flake8",
+    "mccabe",
+    "pyflakes",
+    "pylint",
+    # Security Tools
```

Comment:
I'd put `codespell` under formatting as it's not a security tool.

Reply:
Addressed in https://github.com/tweag/FawltyDeps/pull/365/commits/939043f6fe59516cda60b0d13227d77f50fb114b.